Makefile

```
@@ -5,9 +5,9 @@ AWESTRUCT_CONFIG=--source-dir=content --output-dir=$(OUTPUT_DIR)
 ASSETS_DIR ?= $(OUTPUT_DIR)/assets/bower
 FONTS_DIR ?= $(OUTPUT_DIR)/css/fonts
 VERSION ?= $(BUILD_NUMBER)-$(shell git rev-parse --short HEAD)
-GITHUB_USER ?= $(USER)
+GITHUB_USERNAME ?= $(shell git config --get remote.origin.url | cut -d '/' -f 4)
```

The variable name is used by the person calling `make user-site` to adjust the value so that they don't need to change the script.  By changing the name of the variable, they will be broken until they detect that the variable name has changed.

The cut command does not work for my repository because I've cloned with ssh rather than with https.  I prefer ssh because I already manage private keys for other purposes.  I believe many Jenkins developers are the same.  This change will disrupt those developers who used the rudimentary GitHub pages facility and had a user name that matches their GitHub ID.
```suggestion
GITHUB_USER ?= $(shell git config --get remote.origin.url | cut -d '/' -f 4)
```